src/ui/i18n/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "CORE-WELCOME_MESSAGE": "<strong>Kibana</strong> <small>is loading. Give me a moment here. I'm loading a whole bunch of code. Don't worry, all this good stuff will be cached up for next time!</small>",
+  "CORE-WELCOME_ERROR": "Kibana did not load properly. Check the server output for more information."
+}
```

Comment:
Let's put this file in the kibana plugin directory, since it's the Kibana plugin that registers it with the i18n plugin.